woocommerce/woocommerce-gateway-stripe
# Allow to enable/disable payment methods irrespective of currency requirement

Fixes #2970 

When the store currency is different from the supported currencies of a payment method, we used to show a currency requirement pill beside the payment method name. This was changed in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/2824 as per the new design at that time and we removed the ability to enable/disable a payment method when store currency is not supported by the payment method.

This caused inconvenience for the store owners using multi-currency. Previously they could enable/disable any method irrespective of the store currency. With multi-currency enabled, the options on the checkout page depend on the shopper's selected currency. 

Hence we are bringing back the `Pill` and allowing to enable/disable payment methods irrespective of currency requirement.

### Design 
xVAk98kbRKUByVTrhPc98g-fi-1063%3A33637

Note: The tooltip design is not handled in this PR.

## Testing instructions
- Go to **WooCommerce > Settings > General** and select MXN as the store currency.
- Go to the Stripe settings page and check the payment method list.
- All the payment methods except Card and OXXO should have the new currency requirement pill. 
- You should be able to enable/disable any payment method irrespective of having the currency requirement pill.

**Before**

![before currency](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/9b5a8df7-6483-467a-a03d-03b911ae4711)


**After**
![currency notice](https://github.com/woocommerce/woocommerce-gateway-stripe/assets/33387139/d90b12d6-320b-40cf-a573-c97600cb2927)
